docs/source/deployguide/ca-deploy-topology.md

```diff
@@ -0,0 +1,58 @@
+
+Recall that a Fabric CA performs the following functions on a blockchain network:
+- Registration of identities, or connect to LDAP as the user registry.
+- Issuance of Enrollment Certificates (ECerts). Enrollment is a process whereby the Fabric CA issues a certificate key-pair, comprised of a signing certificate and a private key that forms the identity. The private and public key are first generated locally by the Fabric CA client, and then the public key is sent to the CA which returns an encoded certificate, the signing certificate.
+- Certificate renewal and revocation.
+
+You have the opportunity to customize the behavior of these functions. The first time the CA is started, it looks for a [fabric-ca-server-config.yaml
+file](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/serverconfig.html) which contains the CA configuration parameters. If the file is not there, a default one is created for you. Before you deploy your CA, this topic provides guidance around the parameters in that file and the decisions you need to make in order to customize the CA according to your use case.
+
+## What CA topology will you use on your network?
+
+The topology of CAs on your network can vary depending on how many organizations participate on the network and how you prefer to administer your CAs.
+
+### How many CAs are required?
+
+Before configuring a CA, you need to understand how many CAs are required for your network. If you read the Deployment Process Overview, you'll recall that it is recommended that you deploy two CAs per organization, an enrollment CA and a TLS CA. TLS communications are required for any production network to secure communications between nodes in the organization. Thus, it is the TLS CA that issues those TLS certificates. The enrollment CA on the other hand is used to generate organization and node identities. Also, because this is a _distributed ledger_, the ordering service should not be part of the same organization as the peers, so you will need separate organizations (and therefore CAs) for your peer organizations and ordering service organization. And if multiple organizations contribute nodes to an ordering service, each ordering node would have its own organization CA.
+
+#### Why is a separate TLS server recommended?
+
+A separate TLS server provides an independent chain of trust just for securing communications. Most organizations prefer that the TLS communications are secured by separate crypto material -- from a different root, either from a separate Fabric TLS CA or another external Certificate Authority.
```

Comment:
It may be worth mentioning that you can complete a TLS handshake with a root TLS cert. If you use the same CA, then you can in essence complete a TLS handshake with any root cert!

Reply:
But the whole point of this paragraph is that we are not recommending that you do that in a Production environment.  I'd prefer not to suggest that.

Reply:
Exactly, that would be a bad thing. that would be a reason to use a separate TLS CA.